src/Widgets/HeaderBar.vala

```diff
@@ -47,38 +44,25 @@ public class Akira.Widgets.HeaderBar : Gtk.Box {
     public signal void new_window ();
 
     public HeaderBar () {
-        Object (orientation: Gtk.Orientation.VERTICAL, toggled: true);
+        Object (toggled: true);
     }
 
     construct {
-        headerbar = new Gtk.HeaderBar ();
-
-        headerbar.set_title (APP_NAME);
-        headerbar.set_show_close_button (true);
-
-        pack_start (headerbar, true, true, 0);
-
-        toolbar = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0);
-        toolbar.margin_start = 15;
-        toolbar.margin_end = 15;
-        toolbar.margin_top = 5;
-        toolbar.margin_bottom = 15;
+        set_title (APP_NAME);
+        set_show_close_button (true);
 
-        Gtk.Menu menu_items = new Gtk.Menu ();
+        var menu_items = new Gtk.Menu ();
```

Comment:
This is just a code style thing. It's up to you if you want to use elementary code style or not, of course

Reply:
Yes, definitely I want to follow elementary code style.
I actually can't find it in the documentation if this is a good or bad practice, or it doesn't matter at all.
Thanks

Reply:
As far as I know, there is no functional difference. It’s just “implicitly typed”. So basically the Vala compiler infers the type from what’s on the other side of the `=` symbol. It just saves some characters and personally I find the highlighting makes it easier to find when you declare variables :)